src/main/java/core/basesyntax/service/Convert.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+
+public interface Convert {

```

Comment:
verbs are suitable for method naming, interfaces should be nouns/adjectives
```suggestion
public interface TransactionParser {

```